### PR TITLE
Update dependencies example

### PR DESCRIPTION
**Pull request by patsy-issa**
The following PR updates the following:

- Ember infrastructure
- Convert route to use native class

### CODE REVIEW

File: examples/dependencies/app/components/my-chart.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
-import Component from '@ember/component';
+import Component from "@ember/component";
```
Comment (marcoow) [inline examples/dependencies/app/components/my-chart.js:1]:
```suggestion
import Component from "@glimmer/component";
```

File: examples/dependencies/app/components/my-chart.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 export default Component.extend({
```
Comment (marcoow) [inline examples/dependencies/app/components/my-chart.js:3]:
```suggestion
export default class MyChartComponent extends Component {
```

File: examples/dependencies/app/components/my-chart.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
   didRender() {
     this.highcharts.chart(this.element, {
```
Comment (marcoow) [inline examples/dependencies/app/components/my-chart.js:5]:
```suggestion
    this.args.highcharts.chart(this.element, {
```

File: examples/dependencies/app/components/my-chart.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
-import Component from '@ember/component';
+import Component from "@ember/component";
 
 export default Component.extend({
   didRender() {
```
Comment (marcoow) [inline examples/dependencies/app/components/my-chart.js:4]: This doesn't exist anymore in Glimmer components and should use a modifier instead.

**Review by marcoow — COMMENTED** [review]
I think we should convert the component to a Glimmer component (I assume that would require adding https://github.com/emberjs/ember-render-modifiers as well but I think that's better than showing old components here). Also can you add an acceptance test?

File: examples/dependencies/tests/acceptance/smoke-test.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  test("visiting /", async function (assert) {
+    await visit("/");
+    let faIcons = document.querySelectorAll(".font-awesome-paragraph svg");
+    assert.equal(faIcons.length, 3, "it renders the fa icons");
+
+    assert
+      .dom("[data-highcharts-chart]")
+      .exists("it initialises the highcharts chart");
+  });
+});
```
Comment (marcoow) [inline examples/dependencies/tests/acceptance/smoke-test.js:17]: Should there be a test for the chart as well in case that fails for some reason?

Comment (patsy-issa) [reply examples/dependencies/tests/acceptance/smoke-test.js:17]: `data-highcharts-chart` is added by highcharts when it initialises on the element do you think this is sufficient?

Comment (marcoow) [reply examples/dependencies/tests/acceptance/smoke-test.js:17]: Yes, seems good to me - I think we really only need a smoke test that breaks if this doesn't work at all for some reason 👍

**Review by marcoow — APPROVED** [review]

**Pull request merged**